### ISSUE TITLE
[BUG] AutoTrain advanced CLI: error: unrecognized arguments: --gradient-checkpointing

### ISSUE THREAD

**Issue by Shubham-253** [huggingface/autotrain-advanced#515]
### Prerequisites

- [X] I have read the [documentation](https://hf.co/docs/autotrain).
- [X] I have checked other issues for similar problems.

### Backend

Colab

### Interface Used

UI

### UI Screenshots & Parameters

import os

#@markdown ---
#@markdown #### Project Config
project_name = 'shirts' # @param {type:"string"}
model_name = 'stabilityai/stable-diffusion-xl-base-1.0' # @param ["stabilityai/stable-diffusion-xl-base-1.0", "runwayml/stable-diffusion-v1-5", "stabilityai/stable-diffusion-2-1", "stabilityai/stable-diffusion-2-1-base"]
prompt = 'photo of sks shirt' # @param {type: "string"}

#@markdown ---
push_to_hub = True # @param ["False", "True"] {type:"raw"}
hf_token = "hf_*****" #@param {type:"string"}
repo_id = "****/*****" #@param {type:"string"}

#@markdown ---
#@markdown #### Hyperparameters
learning_rate = 1e-4 # @param {type:"number"}
num_steps = 500 #@param {type:"number"}
batch_size = 1 # @param {type:"slider", min:1, max:32, step:1}
gradient_accumulation = 4 # @param {type:"slider", min:1, max:32, step:1}
resolution = 1024 # @param {type:"slider", min:128, max:1024, step:128}
use_8bit_adam = False # @param ["False", "True"] {type:"raw"}
use_xformers = False # @param ["False", "True"] {type:"raw"}
use_fp16 = True # @param ["False", "True"] {type:"raw"}
train_text_encoder = False # @param ["False", "True"] {type:"raw"}
gradient_checkpointing = True # @param ["False", "True"] {type:"raw"}

os.environ["PROJECT_NAME"] = project_name
os.environ["MODEL_NAME"] = model_name
os.environ["PROMPT"] = prompt
os.environ["PUSH_TO_HUB"] = str(push_to_hub)
os.environ["HF_TOKEN"] = hf_token
os.environ["REPO_ID"] = repo_id
os.environ["LEARNING_RATE"] = str(learning_rate)
os.environ["NUM_STEPS"] = str(num_steps)
os.environ["BATCH_SIZE"] = str(batch_size)
os.environ["GRADIENT_ACCUMULATION"] = str(gradient_accumulation)
os.environ["RESOLUTION"] = str(resolution)
os.environ["USE_8BIT_ADAM"] = str(use_8bit_adam)
os.environ["USE_XFORMERS"] = str(use_xformers)
os.environ["USE_FP16"] = str(use_fp16)
os.environ["TRAIN_TEXT_ENCODER"] = str(train_text_encoder)
os.environ["GRADIENT_CHECKPOINTING"] = str(gradient_checkpointing)

![image_2024-02-17_120654012](https://github.com/huggingface/autotrain-advanced/assets/31141840/e6666f20-bfec-4369-a414-e307d333c708)

### Error Logs

usage: autotrain <command> [<args>]
AutoTrain advanced CLI: error: unrecognized arguments: --gradient-checkpointing

**Comment by abhishekkrthakur** [comment]
im still not sure what this is from. the colab in readme doesnt have any such arguments. please follow official readme and code :)
